frontend/packages/local-storage-operator-plugin/src/components/local-volume-set/types.ts

```diff
@@ -0,0 +1,32 @@
+import { IRow } from '@patternfly/react-table';
+import { NodeKind } from '@console/internal/module/k8s';
+
+export type NodeTableRow = {
+  cells: IRow['cells'];
+  selected: IRow['selected'];
+  props: {
+    data: NodeKind;
+    uid: string;
+  };
+};
+
+export type RowUIDMap = {
+  [key: string]: NodeTableRow;
+};
+
+export type LocalVolumeSetKind = {
+  apiVersion: string;
+  kind: string;
+  metadata: { name: string };
+  spec: {
+    nodeSelector?: {
+      nodeSelectorTerms: [
+        {
+          matchExpressions: [{ key: string; operator: string; values: string[] }];
+        },
+      ];
+    };
```

Comment:
K8sResourceCommon might handle a few of these fields.

Reply:
Sure :+1: